File: CICS 160 assignment 4/Student.java
public class Student extends Person {
    private int year;
    public Student(String newName, String address, String phone, int year){
        super(newName,address,phone);
        this.year = year;

    }

    void setGraduationYear(int year){
        this.year = year;

    }

    int getGraduationYear(){
        return this.year;
    }

    public String toString(){
        return (super.toString()+("Grad Year=" + Integer.toString(getGraduationYear()))+"\n");


    }

public static void main (String[] args){
    System.out.println("testing Student");
    Student aNewStudentObject = new Student("Jaime", "Amherst", "413-545-8929",2026);
    

    System.out.println(aNewStudentObject.toString());
    }
}

File: assignment 3 CICS160/Student.py
from Person import Person
class Student(Person):
    def __init__(self, newName="none", address="none", phone="999-999-9999", year=9999):
        super().__init__(newName,address,phone)
        self.year = year



    def __str__(self):
         return super().__str__()+"Expected Graduation year:"+ str(self.year)+'\n'

    def getGraduationYear(self):
        return(int(self.year))

    def setGraduationYear (self, year):
        self.year = int(year) 
